docs/src/components/ComponentDoc/ComponentExamples.tsx

```diff
@@ -82,23 +81,24 @@ export default class ComponentExamples extends React.Component<ComponentExamples
   )
 
   private testExamplesStructure(displayName: string, allPaths: string[]): string[] {
-    const examplesPattern = `\.\/\\w*\/${displayName}[\\w\/]*\/\\w+Example`
-    const allExamplesRegExp = new RegExp(`${examplesPattern}[\\w\.]*\.tsx$`)
+    const examplesPattern = `\.\/${displayName}[\\w\/]*\/\\w+Example`
```

Comment:
it seems that there is a problem with the regular expression used here - specifically, with the following part of it:

```
[\w\/]
```

Here it seems that the intent is to match either word character or slash, but the problem is that idle slash at start makes this regex to match `w` character literally.

This one will work correctly:

```
[\w\\]
```

The same problem apparently happens to the rest of the expression, where double slash is used here: `\\w+Example`. Am I missing something?

Reply:
I simplified it to:
```js
const examplesPattern = `\./${displayName}/[\\w/]+Example`
```

`examplesPattern` is not regex expression, it's a regular string. I'm not sure that we need there regular expressions at all.

Reply:
This is much simpler to analyze, thanks!

still there is a problem, unfortunately - and this comes from the fact that slashes are not escaped in the string expression. This is what string expression will be evaluated to: `./displayName/[w/]+Example` (with `displayName`='displayName')

And this expression creates several problems: 
- each of the path delimiter  slashes should be escaped
- dot at the start will match any character (is it our intent, or we would like to match dot literally? could we narrow down the set of characters to match, if so?)

I would see the following string should be used instead:

```js
// here I suppose that intent is to match dot character literally, 
// please, make any necessary corrections if it is not
const examplesPattern = `[.][/]${displayName}[/][\w/]+Example`
```

Please, just verify once again this regular expression before merging these changes. Thank you!